src/app/campaign/nav/campaign-nav.component.ts

```diff
@@ -21,16 +21,33 @@ import { FlightState } from '../store/models';
         <mat-icon color="warn" *ngIf="!statusOk(flight)">priority_high</mat-icon>
       </a>
     </mat-nav-list>
-    <a class="add-flight" [routerLink]="" (click)="createFlight.emit()"><mat-icon>add</mat-icon> Add a Flight</a>
+    <a class="secondary-link" routerLink="" (click)="createFlight.emit()"><mat-icon>add</mat-icon> Add a Flight</a>
```

Comment:
maybe at some point we'll decide to "Save & Duplicate" campaigns in one transaction(?) if they've made changes or not yet persisted the campaign, but at this point it's not allowed